src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItem.java

```diff
@@ -42,7 +42,7 @@
     private Timestamp nextActionDate;
 
     @Type(type = "jsonb")
-    @Column(name = "ocrDataJson", columnDefinition = "jsonb")
+    @Column(name = "ocrData", columnDefinition = "jsonb")
```

Comment:
We should only switch to the new column once the application has started storing OCR data in the right format

Reply:
👍